website/docs/r/data_migration_service.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "Data Migration"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_migration_service"
+sidebar_current: "docs-azurerm-resource-data-migration-service"
+description: |-
+  Manage Azure Service instance.
+---
+
+# azurerm_data_migration_service
+
+Manage Azure Data Migration Service instance.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "northeurope"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name 				   = "example-subnet"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefix       = "10.0.1.0/24"
+}
+
+resource "azurerm_data_migration_service" "example" {
+	name                = "example-dms"
+	location            = azurerm_resource_group.example.location
+	resource_group_name = azurerm_resource_group.example.name
+	virtual_subnet_id   = azurerm_subnet.example.id
+	sku_name            = "Standard_1vCores"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specify the name of the data migration service. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) Name of the resource group in which to create the data migration service. Changing this forces a new resource to be created.
+
+* `virtual_subnet_id` - (Required) The ID of the virtual subnet resource to which the data migration service should be joined. Changing this forces a new resource to be created.
```

Comment:
should we be making this `subnet_id` to match other resources?

Reply:
Yes, my mistake 😥